ant-peers-acquisition/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/maidsafe/autonomi"
 version = "0.5.7"
 
 [features]
+default = ["network-contacts"]
```

Comment:
Maybe we should integrate `ant-peers-acquisition` into `ant-bootstrap`? So we have just one crate related to bootstrap contacts.

Reply:
Cool!